README.md

```diff
@@ -1,132 +1,184 @@
- }
+#import "FISPerson.h"
 
-```
+@interface FISPerson ()
 
-But it gets better! Objective-C allows us to use dot notation to both get and set properties.  I could (and in fact should), write the above code like this: 
+@end
 
-```objc
- - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
- {
-   Cat *myCat = [[Cat alloc] init];
-  
-   myCat.numberOfLegs = @4; //sets number of legs 
-   NSNumber *numLegs = myCat.numberOfLegs; // gets myCat's numberOfLegs and assigns it to a local variable 'numLegs'
- }
+@implementation
+
+@end
 ```
 
-Last, remember that the classes you make are just like any other class such as `NSString` or `NSNumber`. You can add them to arrays like this:
+**Top-tip:** *Don't forget to include the empty parenthesis that conclude the private* `@interface` *section.*
 
-```
-NSMutableArray *myArray = [[NSMutableArray alloc] init];
-Cat *myCat = [[Cat alloc] init];
+Now privately redeclare all four properties as `readwrite` within the private `@interface` section.
 
-[myArray addObject:myCat];
-```
+#### Declare the Methods
+
+Publicly declare the ten following methods in the `.h` header file:
 
-Then we can get the cats the same way we are used to with arrays
+  * `init` — the default initializer,
+  * `initWithName:age:` — a convenience initializer,
+  * `initWithName:age:skills:qualifiedTutor:` — a designated initializer,
 
+  * `celebrateBirthday` which returns an `NSString`,
+
+  * `learnSkillBash` which returns `void`,
+  * `learnSkillXcode` which returns `void`,
+  * `learnSkillObjectiveC` which returns `void`,
+  * `learnSkillObjectOrientedProgramming` which returns `void`,
+  * `learnSkillInterfaceBuilder` which returns `void`, and
+
+  * `qualifyAsTutor` which returns a `BOOL`.
+
+#### Define Initial Methods Implementations
+
+Move to the `.m` implementation file. Utilizing autocomplete to quickly fill in the method name, initially define the methods to:
+
+For the initializers, `init`, `initWithName:age:`, and `initWithName:age:skills:qualifiedTutor:`:
+
+```objc
+self = [super init];
+return self;
 ```
-Cat *retreivedCat = myArray[0];
+For the `celebrateBirthday` and `qualifyAsTutor` methods:
+
+```objc
+return nil;
 ```
+For the five `learnSkill...` methods, provide no implementation.
 
-#### Properties 
-Create the following properties in Person.h.  
+Run the tests with `⌘` `U`. The build should succeed but the tests should fail. Throughout solving the rest of this lab, run the tests frequency to check your work as you compose it.
 
-  1. ```(NSNumber *)height```
-  2. ```(NSNumber *)age```
-  3. ```(NSString *)name```
-  4. ```(BOOL) isFemale```
-  5. ```(NSMutableArray *)friends```
+### II. Implement the Initializers
 
-ex. 
-```objc
- @interface Person : NSObject 
+1 — Write the implementation for the designated initializer `initWithName:age:skills:qualifiedTutor:`. Remember to follow the designated initializer format that 
```

Comment:
Suuper weird to be able to set qualifiedTutor when it's really a calculated property